hello.cpp

```diff
@@ -0,0 +1,46 @@
+#include <thread>
+#include <boost/asio.hpp>
+
+#include "packet.h"
+
+using boost::asio::ip::tcp;
```

Comment:
The functions are a little namespace heavy, a personal preference but I would try to have a
```using std::cout/cerr/endl```
or something at the top of the file to alleviate all the line noise of blah::something::other::thing